### DISCUSSION TITLE
Reconfigure

### DISCUSSION THREAD

**Discussion by McFarTech** [Koenkk/zigbee2mqtt#16840] [General]
HI Guys. 

I am finding more and more that i am having to go in and hit reconfigure on a bunch of plugs. Is there any way i could schedule this to happen once a week? Is there a script or process in order to do this?

**Comment by lux73** [comment]
afaik its not normal if you must reconfigure once a Werk

What problems happen if you do not?

**Reply by McFarTech** [reply]
So i am finding that plugs stop updating their data in the Energy dashboard and where i have Node Red flows that are looking for say a power value to drop below 5 and it never happens. As soon as i reconfigure the devices all is good for a week or two but then have to do it again.

**Comment by fyase** [comment]
> HI Guys.
> 
> I am finding more and more that i am having to go in and hit reconfigure on a bunch of plugs. Is there any way i could schedule this to happen once a week? Is there a script or process in order to do this?

Same issue happen to me

**Comment by bacobas** [comment]
Actually we develop a #backend application that communicate in MQTT with Zigbee2Maqtt platform 
and we need to integrate the “#reconfigure” command ( as the Zigbee2Maqtt frontend application)
in our backend application : 
All that for let us able to send the same command by our application .
How can we do that?
Please help us. 
BR
